k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-pod.yaml

```diff
@@ -218,7 +141,8 @@ spec:
             - /bin/sh
           args:
             - -c
-            - /opt/seagate/cortx/hare/bin/hare_setup start --config yaml:///etc/cortx/cluster.conf
+            - /opt/seagate/cortx/hare/bin/hare_setup start --config yaml:///etc/cortx/cluster.conf;
+              sleep infinity;
```

Comment:
Same question, OK to sleep if the start command failed?

Reply:
Yep, same as above via https://github.com/Seagate/cortx-k8s/pull/115#discussion_r812515699